docs/index.md

```diff
@@ -5,6 +5,7 @@
 <br/><br/>
 <a href="https://github.com/jellyfin/jellyfin"><img alt="GPL 2.0 License" src="https://img.shields.io/github/license/jellyfin/jellyfin.svg"></a>
 <a href="https://github.com/jellyfin/jellyfin/releases"><img alt="Current Release" src="https://img.shields.io/github/release/jellyfin/jellyfin.svg"></a>
+<a href="https://translate.jellyfin.org/engage/jellyfin-web/?utm_source=widget"><img src="https://translate.jellyfin.org/widgets/jellyfin-web/-/svg-badge.svg" alt="Translation status" /></a>
```

Comment:
The badge doesn't load, 404

Reply:
Oh, you got it before I pushed up the fix ;-)